backend/settings.py

```diff
@@ -85,8 +85,17 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
+        'NAME': 'user_data',
+        'ENGINE': 'django.db.backends.mysql',
+        'HOST': '127.0.0.1',
+        'PORT': 3306,
+        'USER': 'root',
+        'PASSWORD': 'lmaozedong',  # change this to env variable maybe
+        'OPTIONS': {
+          'autocommit': True,
```

Comment:
I'm not sure if we'll be using any atomic-transactions, but it might be good to note that this should be switched to `False` if so. Consider reading [this MySQL documentation about autocommit, Commit, and Rollback](https://dev.mysql.com/doc/refman/8.0/en/innodb-autocommit-commit-rollback.html)